src/fstests/runner.py

```diff
@@ -54,7 +54,7 @@ def _schedule_job(self, tarball_node, device_config, tmp):
             }
             params.update(self._plan.params)
             params.update(device_config.params)
-            templates = ['config/runtime',
+            templates = ['src/fstests',
```

Comment:
I suspect we have to keep `config/runtime` and just add a line for `src/fstests`, otherwise any base template in `config/runtime` won't be found.  In practice, at the moment all the base templates are in `/etc/kernelci/runtime` so it works, but still there's no good reason for removing `config/runtime` imho.